### PR TITLE
Removed hardcoded module load NGS_DNA version in startFromVcf.sh

### PR DESCRIPTION
**Pull request by RoanKanninga**

### CODE REVIEW

File: startFromVcf.sh [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 set -e
 set -u
-module load NGS_DNA/3.5.2
 module list
```
Comment (pneerincx) [inline startFromVcf.sh:5]: Replacing the hard coded version number for a variable which comes from a config file or commandline option makes sense to me, but completely removing the `module load` statement makes less sense to me.

Comment (pneerincx) [reply startFromVcf.sh:5]: In the commandline help this is listed:
```
-a   sampleType (DNA or RNA) (default=DNA)
```
but there are several instances of `${EBROOTNGS_DNA}` which will not work for RNA data, right? You could also check if `${EBROOTNGS_DNA}` is defined and if not or empty, exit with an error message that a `module load NGS_DNA` is required.

**Review by pneerincx — CHANGES_REQUESTED** [review]
See inline comment.

**Comment by RoanKanninga** [comment]
It is removed completely because the script is executed via the GavinSA
which is part of the NGS_DNA.. which is loaded on forehand, no need to do a
module load again (We don't do it in the generate_template anymore too )

**Review by pneerincx — APPROVED** [review]

**Pull request merged**